RecoPPS/Local/test/totemT2NewDigi_reco_cfg.py

```diff
@@ -0,0 +1,61 @@
+import FWCore.ParameterSet.Config as cms
+
+process = cms.Process('RECO')
+
+# import of standard configurations
+process.load('Configuration.StandardSequences.Services_cff')
+process.load('FWCore.MessageService.MessageLogger_cfi')
+process.MessageLogger.cerr.threshold = "DEBUG"
+# enable LogDebug messages only for specific modules
+process.MessageLogger.debugModules = ["Totem"]
+
+process.load('Configuration.EventContent.EventContent_cff')
+#process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_cff')
+
+#from Configuration.AlCa.GlobalTag import GlobalTag
+#process.GlobalTag = GlobalTag(process.GlobalTag, 'auto:run2_data', '')
+
+# raw data source
+process.source = cms.Source("NewEventStreamFileReader",
+    fileNames = cms.untracked.vstring(
+        '/store/group/dpg_ctpps/comm_ctpps/TotemT2/RecoTest/run364983_ls0001_streamA_StorageManager.dat',
```

Comment:
hi @smuzaffar can you please move this file somewhere where it's accessible to Jenkins? And also help @oljemark on the syntax how to reach it?

Reply:
Hi @tvami and @smuzaffar . Resuming now after being off with a cold: so can the streamer file be copied there (http://cmsrep.cern.ch/cmssw/download/data/), and if so, by whom?

Reply:
@smuzaffar can you please take care of this?

Reply:
sure @tvami , I need to added it under some data external, so is `data-RecoPPS-Local` the right package for this?

Reply:
yes, I think that's going to be good

Reply:
https://github.com/cms-sw/cmsdist/pull/8501 Pr adds `/store/group/dpg_ctpps/comm_ctpps/TotemT2/RecoTest/run364983_ls0001_streamA_StorageManager.dat` in to `RecoPPS-Local` data package. I guess now you can use `cms.untracked.FileInPath(''RecoPPS/Local/data/run364983_ls0001_streamA_StorageManager.dat)` to access this file

Reply:
Hi, @smuzaffar . I don't see the file there quite yet (in nightly 13_2_X_2023-05-08_2300), but I added the weblink to the file in cmsrep ; hopefully that works on Jenkins, too?

Reply:
https://github.com/cms-sw/cmsdist/pull/8501 PR (which adds this file) was only merged half an hour ago, so file will only be available in tonight's 23h00 IB